first_game.py

```diff
@@ -0,0 +1,42 @@
+print("You're a passenger of a ship that sank near a beautiful tropical island. After reaching the beach you look around and decide on one of three options. Type 1 if you 'set a fireplace', type 2 if you 'sit down and cry', type 3 if you 'go to the jungle to get some food'")
+print()
```

Comment:
instead of using print() to create an empty line, add "\n" to the end.

There are also other ways to achieve this, more on the topic here: https://www.freecodecamp.org/news/python-new-line-and-how-to-python-print-without-a-newline/